src/system/Form/ToggleGroup.stories.tsx

```diff
@@ -0,0 +1,37 @@
+/** @jsxImportSource theme-ui */
+
+import { useState } from 'react';
+
+import { ToggleGroup } from './ToggleGroup';
+
+export default {
+	title: 'Toggle Group',
+	component: ToggleGroup,
+};
+
+export const Default = () => {
```

Comment:
Can you please add documentation for this component? Check https://vip-design-system-components.netlify.app/?path=/docs/tooltip--docs as an example on how to do it. 

Also, I believe this component should be a variant of the existing RadioBoxGroup. You can improve the documentation there.